qiskit_nature/drivers/qmolecule.py

```diff
@@ -116,15 +116,34 @@ def __init__(self, filename=None):
         self.nuclear_dipole_moment = None
         self.reverse_dipole_sign = False
 
+        # Custom one and two body integrals
+        self._one_body_integrals = None
+        self._two_body_integrals = None
```

Comment:
I have to say that there is nothing much about this I like. Unlike changing the basic integrals where the qmolecule data is self consistent this now sets these such it will return whatever is set in these. Its not saved in the HDF5, like the rest of the data, that includes the particle hole energy that is extracted, so that would be wrong of reconstituting it from an HDF5 as it would say particle hole energy was extracted but the data return from here would not reflect that.
And as I mentioned the dipoles needed a like treatment so I presume you have in mind the same for those